x-pack/plugins/uptime/server/lib/requests/search/query_context.ts

```diff
@@ -38,15 +39,16 @@ export class QueryContext {
     this.statusFilter = statusFilter;
   }
 
-  async search(params: any): Promise<any> {
-    return this.callES.search({ body: params.body });
+  async search<TParams>(params: TParams) {
+    return this.callES.search(params);
   }
 
   async count(params: any): Promise<any> {
-    return this.callES.count(params);
+    const { body } = await this.callES.count(params);
+    return body;
```

Comment:
This is the regression that is fixed. It had no effect on functionality, only small performance hit.